huaweicloud/services/acceptance/gaussdb/resource_huaweicloud_gaussdb_opengauss_instance_test.go

```diff
@@ -79,6 +79,64 @@ func TestAccOpenGaussInstance_basic(t *testing.T) {
 	})
 }
 
+func TestAccOpenGaussInstance_prepaid(t *testing.T) {
+	var (
+		instance     instances.GaussDBInstance
+		resourceName = "huaweicloud_gaussdb_opengauss_instance.test"
+		rName        = acceptance.RandomAccResourceNameWithDash()
+		password     = fmt.Sprintf("%s@123", acctest.RandString(5))
+		newPassword  = fmt.Sprintf("%sUpdate@123", acctest.RandString(5))
+	)
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&instance,
+		getOpenGaussInstanceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
```

Comment:
better to add a specific precheck for opengauss, it costs too much

Reply:
The precheck has been added:
```
make testacc TEST='./huaweicloud/services/acceptance/gaussdb' TESTARGS='-run=TestAccOpenGaussInstance_prepaid'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb -v -run=TestAccOpenGaussInstance_prepaid -timeout 360m -parallel 4
=== RUN   TestAccOpenGaussInstance_prepaid
=== PAUSE TestAccOpenGaussInstance_prepaid
=== CONT  TestAccOpenGaussInstance_prepaid
    acceptance.go:265: Do not allow expensive testing
--- SKIP: TestAccOpenGaussInstance_prepaid (0.00s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   0.060s
```